### PR TITLE
Basic callback method stuff

### PR DESCRIPTION
**Pull request by dicene**
Builds a way to add a releaseCallback as a method to the Geyser.Label class

### CODE REVIEW

**Comment by dicene** [comment]
@vadi2 and @SlySven Got a few questions on some of how Geyser works. Two pieces of code that are already here that I'm not really sure about...

**Comment by dicene** [comment]
The code in this first commit seems to work for me, although I have yet to test with custom arguments being passed.

File: lua/geyser/GeyserLabel.lua [hunk 1]
```diff
@@ -77,8 +77,17 @@ end
 -- @param ... Parameters to pass to the function. Must be strings or numbers.
 function Geyser.Label:setClickCallback (func, ...)
    setLabelClickCallback(self.name, func, ...)
-   self.callback = func
-   self.args = {...}
+   self.clickCallback = func
+   self.clickArgs = {...}
+end
+
+--- Sets a callback to be used a mouse click is released over this label.
```
Comment (vadi2) [inline lua/geyser/GeyserLabel.lua]: I think "when" is missing here

**Comment by vadi2** [comment]
On nestable - as you and I know now, it's part of the flyout labels: http://wiki.mudlet.org/w/Manual:Geyser#Flyout_Labels.

**Comment by vadi2** [comment]
On sharing args - it just allows you to hook up the same function with the same args to multiple callbacks then. Don't see it as an issue to let someone do that if they wish - would be useful bin debugging, for example.

**Comment by vadi2** [comment]
onEnter - it calls setLabelOnEnter. me: refers to the label instance itself.

Hope that helps! Looking forward to merging this, will solve #19.

**Comment by vadi2** [comment]
Thanks for the improvement @dicene!

**Pull request merged**